buildtest/buildsystem/base.py

```diff
@@ -208,31 +207,29 @@ class BuilderBase:
        any kind of builder.
     """
 
-    def __init__(self, name, recipe_config, config_file=None, testdir=None):
-        """initiate a builder base. A recipe configuration (loaded) is required.
-           this can be handled easily with the BuildConfig class:
+    def __init__(self, name, recipe, buildspec=None, testdir=None):
+        """Initiate a builder base. A recipe configuration (loaded) is required.
+           this can be handled easily with the BuildspecParser class:
 
```

Comment:
bc originally referred to BuildConfig so if this is now BuildParser it should be something like bp